AirCasting/Utils/Strings.swift

```diff
@@ -305,300 +439,501 @@ struct Strings {
+                                                                 comment: "")
+        static let contentViewText_4Mobile: String = NSLocalizedString("Move to your starting location, confirm your location is accurate on the map, then press the start recording button below.",
+                                                                       comment: "")
+        static let startRecording: String = NSLocalizedString("Start recording",
+                                                              comment: "")
+    }
+    
     enum ChooseCustomLocationView {
-        static let sessionLocation: String = LocalizedStringKey("Session location").stringKey
-        static let titleLabel: String = LocalizedStringKey("Search the address and adjust the marker to indicate an exact placement of Your AirBeam").stringKey
+        static let sessionLocation: String = NSLocalizedString("Session location",
+                                                               comment: "")
+        static let titleLabel: String = NSLocalizedString("Search the address and adjust the marker to indicate an exact placement of Your AirBeam",
+                                                          comment: "")
     }
-
+    
     enum MainTabBarView {
-        static let homeIcon: String = LocalizedStringKey("home").stringKey
-        static let homeBlueIcon: String = LocalizedStringKey("bluehome").stringKey
-        static let plusIcon: String = LocalizedStringKey("plus").stringKey
-        static let plusBlueIcon: String = LocalizedStringKey("blueplus").stringKey
-        static let settingsIcon: String = LocalizedStringKey("settings").stringKey
-        static let settingsBlueIcon: String = LocalizedStringKey("bluesettings").stringKey
-        static let loggingOut: String = LocalizedStringKey("Logging out, please wait...").stringKey
-    }
-
+        static let homeIcon: String = NSLocalizedString("home",
+                                                        comment: "")
+        static let homeBlueIcon: String = NSLocalizedString("bluehome",
+                                                            comment: "")
+        static let plusIcon: String = NSLocalizedString("plus",
+                                                        comment: "")
+        static let plusBlueIcon: String = NSLocalizedString("blueplus",
+                                                            comment: "")
+        static let settingsIcon: String = NSLocalizedString("settings",
+                                                            comment: "")
+        static let settingsBlueIcon: String = NSLocalizedString("bluesettings",
+                                                                comment: "")
+        static let loggingOut: String = NSLocalizedString("Logging out, please wait...",
+                                                          comment: "")
+    }
```

Comment:
we don't want strings used to name button icons here at all

`HomeIcon`/`PlusIcon`/`SettingsIcon`